keyvi/include/keyvi/dictionary/dictionary_merger.h

```diff
@@ -49,8 +49,26 @@
 namespace keyvi {
 namespace dictionary {
 
+/**
+ * Exception class for dictionary merger
+ */
+
+struct merger_exception : public std::runtime_error {
+  using std::runtime_error::runtime_error;
+};
+
 template <class PersistenceT, class ValueStoreT = fsa::internal::NullValueStore>
 class DictionaryMerger final {
+  using GeneratorAdapter = fsa::GeneratorAdapterInterface<PersistenceT, ValueStoreT>;
+
+ public:
+  struct MergeStats {
+    MergeStats() : number_of_keys_(0), deleted_keys_(0), updated_keys_(0) {}
+    size_t number_of_keys_;
+    size_t deleted_keys_;
+    size_t updated_keys_;
```

Comment:
As we are defining `default constructor` which only initializes members with zeros, we can use `default member initializers`  instead.
```
size_t number_of_keys_ = 0;
```